VIC_RBM/model_run/result_analysis/scripts/plot_flow_future.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/local/anaconda/bin/python
+
+# This script plots different routed flow results together
```

Comment:
Maybe more description of contact information/dates (though maybe not because it's on github and thus tied to you?)